docs/getting-started.md

```diff
@@ -4,74 +4,119 @@ sidebar_position: 2
 
 # Getting started
 
-Here you will find instructions on how to add OpenTelemetry Agent with Mesmer extension to your application and make it export metrics in a Prometheus format.
+Here you will find instructions on how to add OpenTelemetry Agent with Mesmer extension to your application and make it export metrics to your chosen backend service.
 
-In addition to OpenTelemetry components Mesmer consists of two parts:
+OpenTelemetry supports exporting telemetry data with multiple different protocols. You will find the list of exporters and their configuration options [here](https://github.com/open-telemetry/opentelemetry-java/blob/main/sdk-extensions/autoconfigure/README.md#exporters).
 
-- **Akka Extension** - that runs in the background and is responsible for exporting the metrics to your chosen backend
-- **OpenTelemetry Agent Extension** - that instruments Akka classes to expose metrics for the Extension
+This guide presents steps for Prometheus and OTEL (native OpenTelemetry protocol) exporters.
 
-Both parts need to be included in the application for Mesmer to work.
+## Basic setup
 
-## Step by step guide
+1. Download [opentelemetry-javaagent.jar](https://github.com/open-telemetry/opentelemetry-java-instrumentation/releases/download/v1.12.0/opentelemetry-javaagent.jar) from `opentelemetry-java` Releases.
```

Comment:
```suggestion
1. Download [opentelemetry-javaagent.jar](https://github.com/open-telemetry/opentelemetry-java-instrumentation/releases/download/v1.13.1/opentelemetry-javaagent.jar) from `opentelemetry-java` Releases.
```

That's the highest version of opentelemetry we use in our dependencies. I use it on my machine as well

Reply:
:+1: